server/entity/User.ts

```diff
@@ -236,11 +236,9 @@ export class User {
     const movieDate = new Date();
     if (movieQuotaDays) {
       movieDate.setDate(movieDate.getDate() - movieQuotaDays);
-    } else {
-      movieDate.setDate(0);
```

Comment:
Why was this removed?

Reply:
I'm not sure it matters since you can't set the interval to 0 anyways?

Reply:
It's not the interval though? It's to return the all-time request count if there is no quota.

Though, agreed, it doesn't really matter since we don't actually use that info in any way.

Reply:
Yeah, but setting the date to 0 does nothing? It will return the same value as the default we set with `new Date()`?

Reply:
Oh true, you're right 🤔